### ISSUE TITLE
jQuery support?

### ISSUE THREAD

**Issue by gwarser** [gagarine/no-transition#5]
Animations can be disabled by setting `jQuery.fx.off` to `true`

https://api.jquery.com/jquery.fx.off/

**Comment by gagarine** [comment]
Ohhh that's interesting. I will do some test ASAP. Thanks for the tip.

Of course, if you want to do a pull requests, you are welcome.

**Comment by gagarine** [comment]
Ok I did a test with https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs/executeScript but this run in some kind of isolated context.

So I think the way to go is to insert a script element in the page, execute it, then remove the element.

**Comment by gagarine** [comment]
Curious also about http://getbootstrap.com/javascript/#transitions
`$.support.transition = false`

Bootstrap use jQuery, but perhaps they also have their own transitions that would be nice to disable at the JS level.

**Comment by NiklasGollenstede** [comment]
> but this run in some kind of isolated context.

`jQuery.wrappedJSObject.fx.off = true;`

see: https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Content_scripts#Accessing_page_script_objects_from_content_scripts
Note that using `wrappedJSObject` will probably flag your extension for admin review. And also only works in Firefox(-based) browsers.

**Comment by sergeevabc** [comment]
@gagarine, have you abandoned us?

**Comment by gagarine** [comment]
Yes sadly yes. I'm not using the plugin myself anymore as 1) web devs to got tired of this gimmick 2) Firefox handle animation a lot better in term of CPU usage.

I may return to it when I got some free time.  I will accept pull requests.

**Comment by ssokolow** [comment]
I still use it to disable things like the scrolling backgrounds at https://www.humblebundle.com/ and I wish I had time to write a PR to make it toggle things per-page because it causes "freeze up the entire tab"-level fatal breakages in situations like attempting to scroll down to the bottom in a GitHub Gist code editor widget.
